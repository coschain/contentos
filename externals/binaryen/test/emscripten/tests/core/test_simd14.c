#include <stdio.h>
#include <xmmintrin.h>

void __attribute__((noinline)) foo(__m128 vec, __m128 &outLength)
{
    outLength = vec;
}

int main()
{
    puts("before");
    __m128 x;
    foo(_mm_set1_ps(1.f), x);
    printf("%.2f", x[0]);
}

