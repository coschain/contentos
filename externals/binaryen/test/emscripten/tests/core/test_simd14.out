before
1.00
